src/base/SExprTree.zig

```diff
@@ -307,20 +310,32 @@ fn toStringImpl(self: *const SExprTree, node: Node, writer_impl: anytype, indent
 
             var first = true;
             for (range.begin..range.attrs_marker) |i| {
+                const child = self.children.items[i];
+                // Skip BytesRange nodes when include_linecol is false
+                if (child == .BytesRange and !include_linecol) {
```

Comment:
Feels strange to need the check in multiple places, but still seems fine.

Reply:
Cause someone theoretically could add. new location and miss this.

Reply:
Removed one of these spots that was technically unnecessary. The other spots (including this one) are important for avoiding trailing whitespace or extra newlines in the output.

Reply:
Yeah. That makes senses....the fun of trailing whitespace. I guess the only way to unify would be if the function called on the child returned whether or not to add the trailing whitespace. But that feels like unneccessary extra change.